src/core/annotation.js

```diff
@@ -3144,6 +3147,10 @@ class ChoiceWidgetAnnotation extends WidgetAnnotation {
     super(params);
 
     const { dict, xref } = params;
+
+    this.indices = dict.get("I");
```

Comment:
Could the entries not possibly be indirect objects, i.e. `Ref`s, since in that case I believe that you want the following to be safe here.
```suggestion
    this.indices = dict.getArray("I");
```